posthog/api/personal_api_key.py

```diff
@@ -37,9 +38,12 @@ class PersonalAPIKeyViewSet(
 ):
     serializer_class = PersonalAPIKeySerializer
     lookup_field = "id"
+    permission_classes = [
+        permissions.IsAuthenticated,
+    ]
```

Comment:
This should already be the default, right?

Reply:
Great point, fixed!